docs/integrations/laravel.rst

```diff
@@ -239,14 +252,14 @@ In the following example, we'll use a middleware:
         public function handle($request, Closure $next)
         {
             if (app()->bound('sentry')) {
-                /** @var \Raven\Client $sentry */
+                /** @var \Raven_Client $sentry */
```

Comment:
I think it is a mistake

Reply:
Yep it is! Fixing it, thanks for spotting it!

Reply:
Umh... Wait a minute... It's like that because I copy-pasted from master, but simply replacing that may not work... The docs need to be fully updated to how the 2.x client will work, so maybe we can postpone that to a later PR... Also, @stayallive knows that Laravel integration way better than me!

Reply:
Will for sure take a look at this very soon!

Reply:
@stayallive ?